resources/cognito_user_pools.go

```diff
@@ -0,0 +1,612 @@
+package resources
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider"
+	"github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func CognitoUserPools() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_cognito_user_pools",
+		Description:  "A container for information about the user pool.",
+		Resolver:     fetchCognitoUserPools,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:        "account_recovery_setting",
+				Description: "Use this setting to define which verified available method a user can use to recover their password when they call ForgotPassword",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveCognitoUserPoolAccountRecoverySetting,
+			},
+			{
+				Name:        "admin_create_user_config_allow_admin_create_user_only",
+				Description: "Set to True if only the administrator is allowed to create user profiles",
+				Type:        schema.TypeBool,
+				Resolver:    schema.PathResolver("AdminCreateUserConfig.AllowAdminCreateUserOnly"),
+			},
+			{
+				Name:        "admin_create_user_config_invite_message_template_email_message",
+				Description: "The message template for email messages",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AdminCreateUserConfig.InviteMessageTemplate.EmailMessage"),
+			},
+			{
+				Name:        "admin_create_user_config_invite_message_template_email_subject",
+				Description: "The subject line for email messages",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AdminCreateUserConfig.InviteMessageTemplate.EmailSubject"),
+			},
+			{
+				Name:        "admin_create_user_config_invite_message_template_sms_message",
```

Comment:
I would rename these columns to a short name based on the meaning they are too long.

Reply:
Shortened field names as suggested.